api/views/group.py

```diff
@@ -13,15 +13,39 @@
 from api.models import Group
 from api.processors import GroupProcessor
 from api.serializers import (
+    DashboardGroupSerializer,
+    DashboardGroupWithStudentsSerializer,
+    GroupReadSerializer,
     GroupWriteSerializer,
-    PublicGroupSerializer,
-    PublicGroupWithStudentsSerializer,
 )
+from api.views.mixins import ReadWriteSerializersMixin
 
 
-class PublicGroupViewSet(viewsets.ReadOnlyModelViewSet[Group], CreateModelMixin):
+class CreateGroupMixin(CreateModelMixin):
```

Comment:
I think it's better to put it to either mixins.py or mixins/group.py, since it's not a view

Reply:
Created a separate package for mixins